packages/immer/src/index.ts

```diff
@@ -10,13 +10,9 @@ export type ImmerPluginConfig = {
 function wrapReducerWithImmer(reducer: Redux.Reducer) {
 	// reducer must return value because literal don't support immer
```

Comment:
Is this mean reducer must return object value because immer don't support primitive values? @semoal

Reply:
If it means this way, we may need to remove this comment. Now immer support returns any value [except the `undefined`](https://immerjs.github.io/immer/return/#producing-undefined-using-nothing).

Reply:
yep, lgtm just remove that line and we're ready to ship